frontend/src/app/login/login.component.html

```diff
@@ -1,27 +1,29 @@
-<div class="mdl-card plone-login">
-  <form #f="ngForm" (ngSubmit)="onSubmit(f.value)">
+<div class="login-form">
 
-    <div [hidden]="!authentication_error"
-             class="error">
+  <form #form="ngForm">
+		<div class="mdl-card mdl-shadow--6dp">
+			<div class="mdl-card__title brand-main--background mdl-color-text--white">
+				<h2 class="mdl-card__title-text">Log in</h2>
+			</div>
+	  	<div class="mdl-card__supporting-text">
+        <mdl-textfield type="text" name="username" id="username" label="username"
+                     [(ngModel)]="username" floating-label>
```

Comment:
Two-way binding (like `[(ngModel)]="username"` here) should be avoid everytime it is not needed (in the present case for instance, the model will never push a different value for the username).
I agree it will not make a huge difference here, but as it is a tutorial, it would be better to use simple binding (with just `ngModel` and we get the values in the form object).
Could you restore the previous implementation?